README.md

```diff
@@ -1,4 +1,42 @@
-## Setup the Database
+# Running the App
+The repository contains Docker configuration which can be used out of the box, however, the app can be run without using
+the docker container. Both way are discussed below:
+
+## 1. Run inside Docker Container
+It is the recommended way, the Docker Image of the app is pre-configured to run the migrations, populate the database and
```

Comment:
I couldn't locate where it is "pre-configured to run the migrations".

Rest of the PR looks good to me. This document can be improved over time of course.

Reply:
Sorry, I have just written what the docker image would contain. Currently, it does not either run the updates or automatically populate the database. Would this be the wrong way?

But `migrations` do not need to wait. I can add them immediately. Thanks

Reply:
I am sorry, I took the readme to describe what has been done; instead of thinking about the doc being work-in-progress on its own.